FaynaTown.Tennis.Planner.Bot.Core/Providers/IFileExtensionProvider.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace FaynaTown.Tennis.Planner.Bot.Core.Providers;
+public interface IFileExtensionProvider {
```

Comment:
Little format refactoring suggestion

```suggestion
﻿namespace FaynaTown.Tennis.Planner.Bot.Core.Providers;
﻿
public interface IFileExtensionProvider {
```

Reply:
Done